src/interfaces/stylesheet/config.py

```diff
@@ -0,0 +1,36 @@
+# UI Palette
+COLOR_01 = "#6C7E8E"        # standard label, background
```

Comment:
Is it possible to rename these to some more representative of what they are. Perhaps something like `MEDIUM_GRAY = '#6C7E8E'` or `COLOR_TEXT_BOLD = '#6C7E8E'` (I think that's what this color is used for)?

Reply:
That's a good suggestion, I am going to take that.